module6/lessons/Week5/NetworkingInTheJobHunt.md

```diff
@@ -0,0 +1,79 @@
+
+Step one is to compile a list of target companies. 
+
+Criteria for a target company:
+1. Company that hires developers like you. The company should meet some of the following criteria:
+    * Hiring remotely or in locations you're interested in
+    * .NET/C#
+    * Has hired Turing grads in the past
+    * Open to hiring developers without CS degrees
+1. The company is hiring for growth oriented roles right now (hiring senior devleopers, hiring HR) Things that signal they are in growth not staying alive. Or currently hiring for entry level roles.
+1. Most of your friends peers have never heard of this company
+1. You have some kind of "In"
+    * Your interests are very aligned
+    * You use/love their product
+    * You know someone who works there (even not in tech)
+    * You are hyper local. Maybe their office is super close to where you live
+        *  Someone an hour outside of Fargo, ND. Found a company in that small town, they hadn't hired someone in like 5 years, but they said "sure, we'll interview you!"
+
+Your instructors are happy to look at your list of potential target companies and give feedback! Shoot for 5 companies!
+
+## 2. Who to Message?
+
+Once you have your list of companies, the next step is to collect people at those companies to message on LinkedIn.
+
+Collect a list of:
+* 1-2 software developers
+* 1-2 managers or team leads
+* 1-2 in recruitment or HR
+
+Quantity over quality here, but if you have something in common with the person even better (e.g. went to the same college even for a short period of time, they also went to a coding school, you have a mutual connection)
+
+## 3. How to Reach Out
+
+Now that you've collected 3-6 people for a particular company, it's time to reach out.
+
+The first step is finding a way to contact each person. This is tougher now that LinkedIn has a limit on custom connection messages.
+
+Here is what I would try:
+1. Use your connection: For Turing Alumni, start with a Slack message. For school alumni, see if your school has an alumni directory. For friends or friends, ask to be put in contact.
+1. Try to find their email: Google the person's name. Does a personal website pop up that includes their email? Do they have a Github profile with an email? Or a LinkedIn with an email listed? Tracy has found that emails have a higher response rate anyway!
+1. Use one of your connection requests with a message.
+
+You might also decide that the $29.99 a month is worth it to send unlimited LinkedIn connections with a message.
+
+## 4. What to Say in your Message
+
+The main focus of your message is two things:
+1. Express your excitement about this particular company. Share the "In" you have that got this company on your list.
+2. Ask if they are open to sharing more about what it's like to work there
+
+## 5. When to follow up
+
+I would follow up if I don't hear a response in a week. Sometimes folks are busy or miss a message, so I think it's only helpful to send a follow-up.
+

```

Comment:
Maybe add something about following up on a different platform?  For example, if they first reach out on slack, maybe followup on slack *and* linkedin (not every alum is still active on slack)